pcli/src/command/tx.rs

```diff
@@ -162,6 +165,24 @@ pub enum TxCmd {
     /// Currently, only zero-fee sweep transactions are implemented.
     #[clap(display_order = 990)]
     Sweep,
+
+    /// Perform an ICS-20 withdrawal.
+    #[clap(display_order = 1000)]
+    Ics20Withdrawal {
+        destination_chain_id: String,
+        destination_chain_address: String,
```

Comment:
Can we make this a `--to` field, like in `tx send`?  I'm also wondering if it would be better to lump the destination chain address together with the destination chain ID, like
```
--to cosmos1grgelyng2v6v3t8z87wu3sxgt9m5s03xvslewd@cosmoshub-4
```
That way, it's harder to accidentally send funds to the wrong chain, because you have to specify a fully-qualified address in one piece